sys_app_29192d90db0123008096a455ca96193d.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?><record_update table="sys_app">
     <sys_app action="INSERT_OR_UPDATE">
         <active>true</active>
+        <can_edit_in_studio>false</can_edit_in_studio>
```

Comment:
Hmm.. Not sure about this, not sure if SN set this automatically, or you did.

Reply:
The only thing I manually changed was the code in the Test Step.